m4_sessions/1804inning/ethics_of_AI.md

```diff
@@ -6,12 +6,42 @@ Ellen Cornelius
 Andy Dymond
 Matt Walker
 
-# Name of Session: 
+# Name of Session: AI Carumba!
 
 ## Session Description (Why should students come to this session):
-
+  Come to discuss and share opinions re: the ever increasing presence of AI in the world and how to handle the ethics of AI in our roles as developers.
+ 
 ## Session Outcomes (What will students learn/leave with?):
+  * Students leave with an opinion on AI
+  * Aware of their ethical line
+  * Know what they can do as individuals in their job to hold their values re: AI
+  * How to handle and be aware of their company's ethics
+  * What to do if their company pushes their ethical line
 
 ## Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+- dynamic presentation, we want students moving around and interacting a lot
+- message instructors to tell cohort to not leave their stuff in classroom
 
+INTRO 5/10min
+  - name, cohort & answer to: "When was your last interaction with artifical intelligence?" or something similar
+  
+CONTEXT 20 min / 5
+  - Automation of labor
+  - History of labor automation
+  - What do we do/have we done with displaced workers?
+    - statistics & resources
+    - discussion
+ 
+ MAIN QUESTION
+ caucusing?
+  - What do we do as developers?
```

Comment:
Be sure to come back to this in closing.  What quesiton do I ask myself?  What if my company isn't working in AI?  Do I care?